src/runtime_src/core/tools/common/ReportMailbox.cpp

```diff
@@ -83,7 +83,11 @@ ReportMailbox::getPropertyTree20202( const xrt_core::device * device,
                                        boost::property_tree::ptree &pt) const

```

Comment:
Copyright date needs to updated (line 2). In this case, a new line needs to be added:
`Copyright (c) 2022 Advanced Micro Devices, Inc.`